domains/Vault/ManageVault/Web/ViewHelpers/VaultShowViewHelper.php

```diff
@@ -79,6 +76,12 @@ public static function upcomingReminders(Vault $vault, User $user): array
             ];
         });
 
+        // this line removes the null values that are added when the contact
+        // is not in the vault (in the method above)
+        $remindersCollection = $remindersCollection->filter(function ($value, $key) {
+            return $value != null;
+        });
+
```

Comment:
@asbiin This is what I've added to fix the bug we had on reminders.

The `remindersCollection` might contain an empty item if the contact is not in the vault (in the method, we say `if contact->vault_id != vault, then return null`.
That creates an entry in the collection with `[ 0 => null]` in that case.
So the view didn't work anymore.
To solve this we need to remove all the entries in the collection that contains null entries.

Reply:
And you can write:
```php
$remindersCollection = $remindersCollection->filter(fn ($value) => $value != null);
```